docs/release-notes/NuGet-6.9.md

```diff
@@ -0,0 +1,110 @@
+
+* Rename no-cache to no-http-cache - [#9180](https://github.com/NuGet/Home/issues/9180)
+
+* VS Package Manager Console should close Text View - [#13104](https://github.com/NuGet/Home/issues/13104)
+
+* Vulnerability indicator shows up on dependent project if parent project has transitive vulnerabilities - [#13068](https://github.com/NuGet/Home/issues/13068)
+
+* Conditional updating when *all*  packages are conditioned is broken - [#13034](https://github.com/NuGet/Home/issues/13034)
+
+* Perf issue in AssetsFileDependenciesDataSource - [#13019](https://github.com/NuGet/Home/issues/13019)
+
+* The `NuGetPackageSearchService.GetDeprecationMetadataAsync` in `NuGet.PackageManagement.VisualStudio` is dead code - [#13007](https://github.com/NuGet/Home/issues/13007)
+
+* Vulnerabilities filter shows incorrectly on “Brower” tab when the default tab was “Browse” instead of “Installed” for the previous opening of solution PM UI   - [#12974](https://github.com/NuGet/Home/issues/12974)
+
+* HTTP 401 after some time in VS - [#12961](https://github.com/NuGet/Home/issues/12961)
+
+* [NuGet.Versioning] SemanticVersion.HasMetadata should indicate that Metadata is not null when true - [#12949](https://github.com/NuGet/Home/issues/12949)
+
+* TelemetryUtility.ToJsonArrayOfTimingsInSeconds returns incorrect json array on locales having comma as decimal separator - [#12915](https://github.com/NuGet/Home/issues/12915)
+
+* Static graph-based restore does not respect Interactive option when loading projects - [#12907](https://github.com/NuGet/Home/issues/12907)
+
+* Vulnerabilities InfoBar link to `Manage NuGet Packages` is truncated - [#12835](https://github.com/NuGet/Home/issues/12835)
+
+* NuGet.Build.Tasks caches CredentialProvider device flow timeout. - [#12540](https://github.com/NuGet/Home/issues/12540)
+
+* "error: Sequence contains no matching element" when listing outdated packages - [#12256](https://github.com/NuGet/Home/issues/12256)
+
+* [Bug]: Process argument string is too long when publishing in Visual Studio with static graph enabled - [#11968](https://github.com/NuGet/Home/issues/11968)
+
+* [Bug]: PM UI cannot uninstall packages in multitargeting projects - [#11914](https://github.com/NuGet/Home/issues/11914)
+
+* When a package is installed in the global packages folder, add details about the package location - [#11447](https://github.com/NuGet/Home/issues/11447)
+
+* NuGet should handle duplicate nomination data better.  - [#8749](https://github.com/NuGet/Home/issues/8749)
+
+[List of commits in this release](https://github.com/NuGet/NuGet.Client/compare/6.9.0.74...6.8.0.131)
+
+### Community contributions
+
+Thank you to all the contributors who helped make this NuGet release awesome!
+
+* [KirillOsenkov](https://github.com/NuGet/NuGet.Client/pull/5494)
+  * [5494](https://github.com/NuGet/NuGet.Client/pull/5494) Control embedding Restore files in binlog
```

Comment:
I have a couple of suggestions:

1. The community contributor name should link to their GitHub profile, rather than the PR link.
2. For bug fixes, we currently follow the format `bug title - {link to bug}`. However, I’m unsure why we use a different convention here, such as `{link to pull-request} PR title`.  Can we make it consistent for the next release?